sources/packages/backend/libs/services/src/queue/queue.module.ts

```diff
@@ -74,9 +78,24 @@ function getQueueModules(): BullModuleAsyncOptions[] {
     imports: [ConfigModule],
     useFactory: async (
       configService: ConfigService,
-    ): Promise<BullModuleOptions> => ({
-      prefix: configService.queuePrefix,
-    }),
-    inject: [ConfigService],
+      queueService: QueueService,
+    ): Promise<BullModuleOptions> => {
+      const queueConfig = await queueService.getQueueConfiguration(queue);
+      let queueOptions: BullModuleOptions = {
+        prefix: configService.queuePrefix,
+      };
+      if (queueConfig.attempts && queueConfig.backoff) {
+        queueOptions = {
+          ...queueOptions,
+          // todo: ann
```

Comment:
Are you still going to work on this TODO?

Reply:
thanks... missed to remove